tasks/configure.yml

```diff
@@ -22,6 +22,20 @@
 - set_fact: archivematica_src_configure_ss_api_key={{ 999999999999999999999 | random | to_uuid | hash('md5') }}
   when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
 
+- name: "Get API Key"
+  become: "yes"
+  command: >
+    mysql --user="{{ archivematica_src_ss_db_user }}"
```

Comment:
Hi @Rayzilt,

When querying SS database we prefer to use dbshell instead of mysql or sqlite3 commands because this way we can include both SS database cases in a row. I have tested the following task and works:
```
- name: "Get SS API Key"
  shell: >
    echo "select key from tastypie_apikey where id in (select id from auth_user where username = '{{ archivematica_src_configure_ss_user }}');"
    | {{ archivematica_src_ss_virtualenv }}/bin/python manage.py dbshell
  args:
    chdir: "{{ archivematica_src_ss_app }}"
    executable: /bin/bash
  environment: "{{ archivematica_src_ss_environment }}"
  delegate_to: "{{ archivematica_src_configure_ss_url|urlsplit('hostname') }}"
  become: yes
  remote_user: "{{ archivematica_src_configure_ss_ssh_user | default('artefactual') }}"
  register: archivematica_src_configure_ss_api_key_temp
  when: "archivematica_src_configure_dashboard|bool and archivematica_src_configure_ss_api_key is undefined"

- set_fact: archivematica_src_configure_ss_api_key="{{ archivematica_src_configure_ss_api_key_temp.stdout }}"
  when: "archivematica_src_configure_dashboard|bool and archivematica_src_configure_ss_api_key_temp.stdout is defined"
``` 
I added a new variable, called `archivematica_src_configure_ss_ssh_user`, that should be the `ansible_ssh_user` you configured on ansible inventory to connect to the SS host. The default value is `artefactual`

Reply:
I've update the PR to the recommended way.